.github/workflows/add-lockdown-label.yml

```diff
@@ -0,0 +1,81 @@
+name: Add Branch Lockdown Label to PRs
+
+on:
+  schedule:
+    # Run daily at 00:00 UTC
+    - cron: '0 0 * * *'
```

Comment:
Nit for what it's worth: GitHub recommends picking a time other than midnight UTC and even suggests picking arbitrary minutes within the hour to avoid resource contention at the most popular time for scheduled workflows to kick off.